packages/frontend/src/lib/WormyEarth/core/Physics.ts

```diff
@@ -29,7 +29,7 @@ class Physics {
   };
 
   public player = (position: Matter.Vector, size: types.Size) => {
-    const body = Matter.Bodies.rectangle(position.x, position.y, size.width, size.height);
+    const body = Matter.Bodies.rectangle(position.x, position.y, size.width, size.height, { density: 1 });
```

Comment:
Create constant for this, `constants.player.density`.
Also, create one for projectile.